soliket/cross_correlation.py

```diff
@@ -0,0 +1,63 @@
+"""
+Simple likelihood for CMB-galaxy cross-correlations using the cobaya
+CCL module. 
+
+First version by Pablo Lemos
+"""
+
+import numpy as np
+from .gaussian import GaussianData, GaussianLikelihood
+import pyccl as ccl
+
+class CrossCorrelationLikelihood(GaussianLikelihood):
+    def initialize(self):
+        self.auto_file: str = 'soliket/data/xcorr_simulated/clgg_noiseless.txt'
```

Comment:
All these properties should be class-level properties or set in the .yaml file for the likelihood (not set within `.initialize()`), and paths should be obtained using `get_resource_filename`, if the data is to be packaged within soliket.